setups/sai_dpu_client_server.json

```diff
@@ -0,0 +1,42 @@
+{"DPU": [
```

Comment:
Please confirm that a setup description file is optional and original approach to run TCs in standalone/client-server mode is supported.